Variables.js

```diff
@@ -17,6 +17,7 @@ const MATERIAL_PLASTIC = 3;
 //Full time
 const VARIABLE_MIN_WEEKLY_VISITS = "minWeeklyVisits";
 const VARIABLE_MIN_WEEKLY_CHORE_TIME = "minWeeklyChoreTime";
+const VARIABLE_MIN_WEEKLY_CHORES = "minWeeklyChores";
```

Comment:
Why if we have a min time which makes more sense because some chores might take longer than others?

Reply:
yeah its from the legacy chore implimentation I was playing... let me run on the new routine for a while and then eventually cleanup...